### ISSUE TITLE
Bundling error with webpack 1

### ISSUE THREAD

**Issue by frontmesh** [matteocng/react-flag-icon-css#45]
I am getting the following when I try to bundle my app with webpack 1. 

I installed all the neceserry packages required for webpack 1 but still it is happening.

Any ideas ? 

### Basic information:

- `react-flag-icon-css` version: latest
- Module bundler: webpack version: 1.13.1
- The output of `node -v`: v 7.2.0

- [x] I can not install `react-flag-icon-css`:
```
export const sizes = ['lg', '2x', '3x', '4x', '5x']
^^^^^^
SyntaxError: Unexpected token export
```
- [x] I am using `yarn` version: v0.24.4
- [x] I am using React Css Modules version: .........

The configuration file of my module bundler:
```js
{
  test: /\.jsx?$/,
  exclude: /node_modules/,
  loader: require.resolve('babel-loader'),
  query: {
    presets: [
      require.resolve('babel-preset-es2015')
    ]
  }
};
```
* * *

**Comment by matteocng** [comment]
Hi @frontmesh, thanks for reporting this issue (`Webpack 1` correctly expects a `CommonJS` module, not an `ES2015` module), I have just published `1.0.17` that restores `Webpack 1` compatibility. Please `yarn upgrade react-flag-icon-css`and if it still doesn't work try a) changing your regex to `/\.(js|jsx)$/` b) adding `require.resolve('babel-preset-react')` to `presets`.

**Comment by frontmesh** [comment]
Hi @matteocng , appreciate the quick response and explanation. 
It has been resolved with new version `1.0.17`.
Thanks!
